docs/setup/install/deb.asciidoc

```diff
@@ -115,6 +122,10 @@ sudo dpkg -i kibana-{version}-amd64.deb
 <1> Compare the SHA produced by `shasum` with the
     https://artifacts.elastic.co/downloads/kibana/kibana-{version}-amd64.deb.sha512[published SHA].
 
+Alternatively, you can download the following package, which includes only 
+Apache 2.0 licensed code: 
```

Comment:
This isn't really accurate. The distributions as a whole are offered under the apache2 license, but the code within them is licensed under many different apache2-compatible licenses, like MIT, BSD, etc.  I think a simple fix here is referring to the license of the distribution itself rather than the code within it, since that's the important detail for the sake of installation. Perhaps:

> Alternatively, you can download the following package, which contains only features that are available under the Apache 2.0 license.